initiator.go

```diff
@@ -19,66 +26,68 @@ type ContainerConfig struct {
 	Tmpfs         map[string]string
 }
 
-var obsoleteAfter float64 = 10 * 60 // in seconds
+var ObsoleteAfter = 10 * time.Minute
```

Comment:
Nice change to `time.Duration` 👍

Reply:
Yep, I'm pretty sure I built this before understanding the duration model in Go :)